backend/apis/parse-docket-pdf.utils.js

```diff
@@ -89,13 +89,80 @@ function parseCharges(lines, sections) {
 }
 
 function parseAccountSummary(lines, sections) {
-  const accounts = [];
+  let subsections = [];
+  let balances = [];
```

Comment:
This should still addressed

Reply:
The balances array is the return array for the parseAccounts function. It has to be initialized outside the addAccounts function so that it can return its modified contents from parseAccounts after addAccounts has been called by the walkSection function. (That's a mouthful.)

Reply:
You can declare variables without initializing them. My comment here is that the `= []` is pointless, since it's always overridden on line 110 no matter what. The `[]` doesn't seem to be used in any way:

```suggestion
  let balances;
```